bots/honkbot.py

```diff
@@ -12,18 +12,21 @@
 class Honkbot(commands.Cog):
     def __init__(self, logger, speedrun_api, google_api, bot=None):
         self.eamuse_maintenance = {
-            "normal": (
+            "daily": (
                 datetime.time(hour=20, tzinfo=pytz.utc),
                 datetime.time(hour=22, tzinfo=pytz.utc),
             ),
             "extended": (
                 datetime.time(hour=17, tzinfo=pytz.utc),
                 datetime.time(hour=22, tzinfo=pytz.utc),
-            ),
-            "us": (
-                datetime.time(hour=12, tzinfo=pytz.utc),
-                datetime.time(hour=17, tzinfo=pytz.utc),
-            ),
+            )
+        }
+
+        self.messages = {
+            'us_games': 'US Servers (DDR White, IIDX Lightning)',
```

Comment:
Tsum Tsum also runs on the US servers, for what it's worth.

Reply:
I also heard SDVX will be shortly as well? This is the text I was given. I don't know if we want to enumerate everything here, or if these were examples, or if this stuff is just known by fans?

Maybe another command that lists the different servers might work.

Reply:
I can definitively say that currently, the US server supports Tsum Tsum, DDR and IIDX Lightning Model. Whether future SDVX releases (Exceed Gear, etc) will be on the US server remains to be seen, but I have also heard that rumor. If Round1 purchases Valkyrie Model cabs, it wouldn't be unheard of for the game to be moved to the US server.

The Japanese server supports all Konami titles. Current titles on Japanese e-amusement at Round1 in the US are DDR Gold, Gitadora, SDVX, IIDX (old school cabs), and DanceRush Stardom.

Reply:
I considered including TsumTsum but, while it’s true, I don’t think anyone actually cares about or plays it.

Anything involving future games running on US servers we will deal with when we know there. It will not be difficult to add another comma list item to the message :)

Reply:
I also agree with tsum tsum not being on there just because who cares lol. Also I generally think having a long string of comma separated titles is pretty ugly and I want to avoid it if possible. I don't know what it would look like as things get bigger but eh. I think this is fine for now but maybe something to keep in mind moving forward.